src/CdrReader.test.ts

```diff
@@ -131,13 +131,15 @@ describe("CdrReader", () => {
   it.each([
     ["float32Array", "float32", [-3.835, 0, Math.PI], 6],
     ["float64Array", "float64", [-3.835, 0, Math.PI], 15],
+    // eslint-disable-next-line @typescript-eslint/no-loss-of-precision
     ["float64Array", "float64", [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, -0.123456789121212121212], 15],
   ])("reads float %s", (getter: string, setter: string, expected: number[], numDigits: number) => {
     const writer = new CdrWriter();
     writeArray(writer, setter as Setter, expected);
 
     const reader = new CdrReader(writer.data);
     const array = reader[getter as ArrayGetter](reader.sequenceLength());
+    expect(array).toBeDefined();
```

Comment:
Added this expect here because was getting a no assertion error on lint. This test always had assertions Lint just didn't recognize them because they're on the function called on line 143. So this new expect() is a just a placeholder to deal with the error referred above